src/displayapp/screens/Navigation.cpp

```diff
@@ -128,29 +128,29 @@ namespace {
  * Navigation watchapp
  *
  */
-Navigation::Navigation(Pinetime::Applications::DisplayApp* app, Pinetime::Controllers::NavigationService& nav)
-  : Screen(app), navService(nav) {
+Navigation::Navigation(Pinetime::Applications::DisplayApp* app, Pinetime::Controllers::NavigationService& nav, Controllers::MotorController& motorController)
+  : Screen(app), navService(nav), motorController {motorController} {
 
   imgFlag = lv_label_create(lv_scr_act(), nullptr);
   lv_obj_set_style_local_text_font(imgFlag, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, &lv_font_navi_80);
   lv_obj_set_style_local_text_color(imgFlag, LV_LABEL_PART_MAIN, LV_STATE_DEFAULT, LV_COLOR_CYAN);
   lv_label_set_text(imgFlag, iconForName("flag"));
-  lv_obj_align(imgFlag, nullptr, LV_ALIGN_CENTER, 0, -60);
+  lv_obj_align(imgFlag, nullptr, LV_ALIGN_CENTER, 0, -78);
```

Comment:
This could be aligned to the top edge. The suggested y offset may need to be tweaked.
```suggestion
  lv_obj_align(imgFlag, nullptr, LV_ALIGN_IN_TOP_MID, 0, 0);
```